mailpoet/assets/css/src/components-plugin/_forms.scss

```diff
@@ -243,3 +243,8 @@ progress::-moz-progress-bar {
 .mailpoet-form-field-disabled {
   cursor: not-allowed;
 }
+
+.mailpoet-password-input-toggle {
+  height: 34px;
```

Comment:
The `Show/Hide` button overflows when used inside the small input components, and it's missing border radius (this one was present also on the `Key Activation` page).

<img width="1519" alt="Screenshot 2025-07-04 at 14 05 33" src="https://github.com/user-attachments/assets/94f3c225-2963-476c-bb29-0da88a0653f0" />

```suggestion
  border-radius: 0 3px 3px 0 !important;
  height: 100%;
```

<img width="907" alt="Screenshot 2025-07-04 at 14 08 00" src="https://github.com/user-attachments/assets/e8ee7f5e-6aef-4b29-ba0a-e678a0b6f025" />